### PR TITLE
#181 partial

### PR DESCRIPTION
**Pull request by teosarca**
see https://github.com/metasfresh/metasfresh-webui/issues/181

### CODE REVIEW

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HULoader.java [hunk 1]
```diff
@@ -64,6 +64,11 @@
 
 public class HULoader
 {
+	public static final HULoader of(final IAllocationSource source, final IAllocationDestination destination)
```
Comment (metas-ts) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HULoader.java]: good, but i propose to deprecate the public constructor (or make it private), to make sure that devs know the *one* way of getting a HULoader instance

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HUProducerDestination.java [hunk 2]
```diff
@@ -45,6 +51,34 @@
  */
 public class HUProducerDestination extends AbstractProducerDestination
 {
+	public static final HUProducerDestination of(final I_M_HU_PI huPI)
```
Comment (metas-ts) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HUProducerDestination.java]: same as for HULoader

Comment (teosarca) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HUProducerDestination.java]: idem

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HULoader.java (hunk 1 above)
Comment (teosarca) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HULoader.java]: agree. The best would be to refactor & deprecate.
btw, are u taking care of this or shall i?

Comment (metas-ts) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HULoader.java]: i can do it now.

**Pull request merged**